packages/api/src/routes/medical/schemas/patient.ts

```diff
@@ -38,3 +38,51 @@ export function schemaUpdateToPatientData(input: PatientUpdate): PatientData {
 export function schemaDemographicsToPatientData(input: Demographics): PatientData {
   return schemaCreateToPatientData(input);
 }
+
+const coverageAssessmentPatientSchema = z.object({
+  dob: z.string(),
+  gender: z.string(),
+  firstname: z.string(),
+  lastname: z.string(),
+  zip: z.string(),
+  city: z.string(),
+  state: z.string(),
+  addressline1: z.string(),
+  addressline2: z.string().optional(),
+  phone1: z.string().optional(),
+  phone2: z.string().optional(),
+  email1: z.string().optional(),
+  email2: z.string().optional(),
+  externalid: z.string().optional(),
+});
+
+export const coverageAssessmentSchema = z.object({
+  patients: coverageAssessmentPatientSchema.array(),
+});
+
+export const coverageAssessmentValidationSchema = z.object({
+  patients: coverageAssessmentPatientSchema
```

Comment:
Maybe this will be answered in the previous comment/thread, but this is odd that we're redefining the schema again w/ optional fields.

Reply:
The idea behind the two endpoints was to tailor the response about how to fix any issues, instead of having to parse zod responses and tie them to patient payloads. That being said, that is possible, just more work :)

Reply:
Done -- I'm simply dumping the zod error into the UI as parsing could be difficult based on what kind of error zod is throwing.

Reply:
Sounds good. It might be useful to check OSS' `zodResponseBody` on `default-error-handler.ts`

Reply:
I hook into this value on internal `title: "Missing or invalid parameters",` so we should be good.